src/scenes/level_scene.py

```diff
@@ -1328,15 +1326,15 @@ def process_entity_action(self, entity: Movable, is_ally: bool) -> None:
         entity -- the entity for which the action should be computed
         is_ally -- a boolean indicating if the entity is an ally or not
         """
-        possible_moves: dict[tuple[int, int], int] = self.get_possible_moves(
+        possible_moves: dict[Position, int] = self.get_possible_moves(
             tuple(entity.position), entity.max_moves
         )
         targets: Sequence[Movable] = (
             self.entities.foes if is_ally else self.players + self.entities.allies
         )
-        allies: Sequence[Movable] = (
-            self.players + self.entities.allies if is_ally else self.entities.foes
-        )
+        # allies: Sequence[Movable] = (
```

Comment:
I assume it's something you planned to fix later on because it's still only a draft PR (I know), but I'm putting this comment as a reminder

Reply:
allies doesn't actually get used by the duel function on line 1194. Seems like something that might get implemented in the future, so I didn't want to get rid of it entirely.

Reply:
Nah feel free to remove not used stuff.

We are using Git for a reason: if some later time we need these changes (because of a feature related to allies) then we could just get back what was removed by the commit.

Commented out code is a code smell, and is harmless to remove when having a good versioning system tool.